base/random.jl

```diff
@@ -1596,6 +1618,30 @@ end
 # each element of A is included in S with independent probability p.
 # (Note that this is different from the problem of finding a random
 #  size-m subset of A where m is fixed!)
+
+"""
+    randsubseq!([rng::AbstractRNG=GLOBAL_RNG,] S, A, p)
+
+Like [`randsubseq`](@ref), but the results are stored in `S`
+(which is resized as needed).
+
+# Examples
+```jldoctest
+julia> b = zeros(4);
+
+julia> a = collect(1:8);
+
+julia> rng = MersenneTwister(1234);
+
+julia> randsubseq!(rng, b, a, 0.5);
+
+julia> b
```

Comment:
@nalimilan [https://github.com/JuliaLang/julia/pull/22324#discussion_r121264661](suggested) to me that in this case, this may be better to show directly the output of the mutating function call, as it is already documented what is mutated. If you want to keep this variable, better to rename it `S`, as in the signature.